src/modules/launcher/Wox.Test/Plugins/WindowsIndexerTest.cs

```diff
@@ -299,11 +299,11 @@ public void LoadContextMenus_MustNotLoadRunAsAdminAndOpenContainingFolder_ForFol
             mockapi.Verify(m => m.GetTranslation("Microsoft_plugin_indexer_open_in_console"), Times.Once());
         }
 
-        [TestCase(0, false, ExpectedResult = false)]
-        [TestCase(0, true, ExpectedResult = true)]
-        [TestCase(1, false, ExpectedResult = true)]
-        [TestCase(1, true, ExpectedResult = true)]
-        public bool DriveDetection_MustDisplayResults_WhenEnhancedModeIsOnOrWhenWarningIsDisabled(int enhancedModeStatus, bool disableWarningCheckBoxStatus)
+        [TestCase(0, false, ExpectedResult = true)]
+        [TestCase(0, true, ExpectedResult = false)]
+        [TestCase(1, false, ExpectedResult = false)]
+        [TestCase(1, true, ExpectedResult = false)]
+        public bool DriveDetection_MustDisplayResults_WhenEnhancedModeIsOnOrWhenWarningIsDisabled(int enhancedModeStatus, bool disableWarningCheckBoxStatus) 
```

Comment:
I think this should be updated to DriveDetection_MustDisplayWarning_WhenEnhacedModeIsOffOrWhenWarningIsNotDisabled

Reply:
Thanks for pointing that out, I completely missed it. However, I think it should be WhenEnhacedModeIsOff**And**WhenWarningIsNotDisabled (And instead of Or). Renamed it accordingly.